src/ray/raylet/node_manager.cc

```diff
@@ -2161,7 +2187,7 @@ void NodeManager::ForwardTask(const Task &task, const ClientID &node_id,
       });
 }
 
-void NodeManager::DumpDebugState() {
+void NodeManager::DumpDebugState() const {
```

Comment:
Do we still need this function? `RecordMetrics` should do also report all the debugging information. @ericl

Reply:
Let's keep it for now.